internal/pkg/manifest/workload_ecs.go

```diff
@@ -131,6 +131,7 @@ type ExposedPort struct {
 	ContainerName string // The name of the container that exposes this port.
 	Port          uint16 // The port number.
 	Protocol      string // Either "tcp" or "udp", empty means the default value that the underlying service provides.
+	Implicit      bool   // Defines if the port is the main port of the container. defaults to false.
```

Comment:
Why is this named `Implicit`?

`Explicit` seemed to make more sense to me 😂  That "this port is explicitly exposed via `image.port` , `sidecar.port`

Reply:
Yeah, I changed it. Also me and Efe decided to call this as `IsContainerDefinedPort`. So updating changes in the upcoming commit.